File: spring/springsecurity/springboot-inmemory-auth/src/main/java/com/javaworm/springbootinmemoryauth/controller/HomeController.java
package com.javaworm.springbootinmemoryauth.controller;

import org.springframework.security.core.Authentication;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
public class HomeController {
	
//	@Autowired
//  private UserDetailsService userService;

    @GetMapping("/")
    public String greeting(Authentication authentication) {
        return "Spring Security In-memory Authentication Example";
    }
}

File: Kotlin/KotlinIntro/src/main/kotlin/com/javaworm/KotlinIntro/controller/HomeController.kt
package com.javaworm.KotlinPractice1.controller

import org.springframework.stereotype.Controller
import org.springframework.ui.Model
import org.springframework.ui.set
import org.springframework.web.bind.annotation.GetMapping

@Controller
class HtmlController {

  @GetMapping("/")
  fun blog(model: Model): String {
    model["title"] = "Hello World from Kotlin"
    return "blog"
  }

}

